.github/workflows/call-build-linux-arm-packages.yml

```diff
@@ -0,0 +1,78 @@
+name: Reusable workflow to build binary packages into S3 bucket
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: The version of Valkey to create.
+        type: string
+        required: true
+      ref:
+        description: The commit, tag or branch of Valkey to checkout for building that creates the version above.
+        type: string
+        required: true
+      build_matrix:
+        description: The build targets to produce as a JSON matrix.
+        type: string
+        required: true
+    secrets:
+      token:
+        description: The Github token or similar to authenticate with.
+        required: true
+      bucket:
+        description: The name of the S3 bucket to push packages into.
+        required: false
+      access_key_id:
+        description: The S3 access key id for the bucket.
+        required: false
+      secret_access_key:
+        description: The S3 secret access key for the bucket.
+        required: false
+
+jobs:
+  build-valkey:
+    # Capture source tarball and generate checksum for it
+    name: Build  package ${{ matrix.distro.target }} ${{ matrix.distro.arch }}
+    runs-on: 'ubuntu-latest'
+    permissions:
+      contents: read
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJSON(inputs.build_matrix) }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.version }}
+      
+      - name: Make Valkey
+        uses: uraimo/run-on-arch-action@v2
+        with:
+          arch: aarch64
+          distro: ${{matrix.distro.target}}
+          install: apt-get update && apt-get install -y build-essential libssl-dev
+          run: make -C src all BUILD_TLS=yes MALLOC=libc
```

Comment:
Why do we need `MALLOC=libc` here?

Reply:
All the references I found on building redis for ARM suggested the `libc` as it is tested more for ARM as compared to `jemalloc`
I would need suggestions, so I can may be change it if we want to.

Reply:
We should use jemalloc.